v_3_7_0/cloudconfig.go

```diff
@@ -13,6 +13,8 @@ import (
 
 const (
 	defaultRegistryDomain = "quay.io"
+	kubernetesImage       = "giantswarm/hyperkube:v1.12.1"
```

Comment:
Where did `v1.11.1-cec4fb8023db783fbf26fb056bf6c76abfcd96cf-giantswarm` come from and why is the image here different?

Reply:
Did you see the title of the PR? :smirk: 

I guess your question is not why this image is different but why the image name format looks different. The `-giantswarm` image was used because we patched the api-server memory leak in `v1.11.1`.

Reply:
The second part answers my question.